### PR TITLE
(BKR-779) skip_test broken in with_puppet_running_on

### PR DESCRIPTION
**Pull request by er0ck**

### CODE REVIEW

**Comment by er0ck** [comment]
retest this please

**Comment by er0ck** [comment]
these all pass locally 🙀

**Comment by er0ck** [comment]
retest this please

File: acceptance/config/puppetgit/acceptance-options.rb [hunk 1]
```diff
@@ -1,4 +1,9 @@
 {
+    :type                 => 'foss',
+    :add_el_extras        => 'true',
+    :is_puppetserver      => false,
+    :puppetservice        => 'puppet.service',
     :pre_suite            => 'acceptance/pre_suite/puppet_git/install.rb',
-    :tests                => 'acceptance/tests/puppet'
-}.merge(eval File.read('acceptance/config/acceptance-options.rb'))
\ No newline at end of file
+    :tests                => 'acceptance/tests/puppet',
```
Comment (er0ck) [inline acceptance/config/puppetgit/acceptance-options.rb]: why are we running all the puppet tests against each puppet install type?  i suppose before we added this one it was how we were testing the install stuff.  feels kinda backwards to me.

**Comment by er0ck** [comment]
argh.  ci jobs don't use the rakefile tasks.  and don't use the options files i just worked so hard to fix to make these tests work.

**Comment by er0ck** [comment]
@puppetlabs-jenkins retest this please

**Comment by er0ck** [comment]
@puppetlabs-jenkins retest this please

**Comment by er0ck** [comment]
hmmm... just osx109:
```
Test Case /var/lib/jenkins/workspace/qe_beaker_intn-sys_vpool-pe372/agent/osx109/master/centos7/pe_acceptance_tests/acceptance/tests/pe_checks/spot_check_lm.rb reported: #<Beaker::DSL::Outcomes::FailTest: 
    h49cu21fd9rvmhh.delivery.puppetlabs.net is not found in LM!
    >
```

**Comment by er0ck** [comment]
seems unrelated?
```
Verifying LM availability
    
    r6arg84ns8eovpm.delivery.puppetlabs.net (centos7-64-1) 17:00:35$ curl --tlsv1 --insecure --cert /etc/puppetlabs/puppet/ssl/certs/r6arg84ns8eovpm.delivery.puppetlabs.net.pem --key /etc/puppetlabs/puppet/ssl/private_keys/r6arg84ns8eovpm.delivery.puppetlabs.net.pem -H 'X-Proxy-User: system' https://r6arg84ns8eovpm.delivery.puppetlabs.net:4435/live-management/lm/nodes
        % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                       Dload  Upload   Total   Spent    Left  Speed
        0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0        0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0        0     0    0     0    0     0      0      0 --:--:--  0:00:02 --:--:--     0        0     0    0     0    0     0      0      0 --:--:--  0:00:03 --:--:--     0        0     0    0     0    0     0      0      0 --:--:--  0:00:04 --:--:--     0        0     0    0     0    0     0      0      0 --:--:--  0:00:05 --:--:--     0        0     0    0     0    0     0      0      0 --:--:--  0:00:06 --:--:--     0        0     0    0     0    0     0      0      0 --:--:--  0:00:07 --:--:--     0        0     0    0     0    0     0      0      0 --:--:--  0:00:08 --:--:--     0        0     0    0     0    0     0      0      0 --:--:--  0:00:09 --:--:--     0      100   102  100   102    0     0     11      0  0:00:09  0:00:09 --:--:--    25
      {"id":"c255961a7091cfb42b8aac9008d4c98c","nodes":[{"name":"r6arg84ns8eovpm.delivery.puppetlabs.net"}]}    
    r6arg84ns8eovpm.delivery.puppetlabs.net (centos7-64-1) executed in 9.22 seconds
    
    r6arg84ns8eovpm.delivery.puppetlabs.net (centos7-64-1) 17:00:44$ puppet agent --configprint node_name_value
      r6arg84ns8eovpm.delivery.puppetlabs.net
    
    r6arg84ns8eovpm.delivery.puppetlabs.net (centos7-64-1) executed in 0.43 seconds
    
    h49cu21fd9rvmhh.delivery.puppetlabs.net (osx109-64-2) 17:00:44$ puppet agent --configprint node_name_value
      h49cu21fd9rvmhh.delivery.puppetlabs.net
    
    h49cu21fd9rvmhh.delivery.puppetlabs.net (osx109-64-2) executed in 0.89 seconds
    
    h49cu21fd9rvmhh.delivery.puppetlabs.net (osx109-64-2) 17:00:45$ puppet agent --configprint node_name_value
      h49cu21fd9rvmhh.delivery.puppetlabs.net
    
    h49cu21fd9rvmhh.delivery.puppetlabs.net (osx109-64-2) executed in 0.89 seconds
    Warning: 
    h49cu21fd9rvmhh.delivery.puppetlabs.net is not found in LM!
    
    /var/lib/jenkins/workspace/qe_beaker_intn-sys_vpool-pe372/agent/osx109/master/centos7/lib/beaker/dsl/outcomes.rb:42
    /var/lib/jenkins/workspace/qe_beaker_intn-sys_vpool-pe372/agent/osx109/master/centos7/pe_acceptance_tests/acceptance/tests/pe_checks/spot_check_lm.rb:20
    /var/lib/jenkins/workspace/qe_beaker_intn-sys_vpool-pe372/agent/osx109/master/centos7/pe_acceptance_tests/acceptance/tests/pe_checks/spot_check_lm.rb:19
    /var/lib/jenkins/workspace/qe_beaker_intn-sys_vpool-pe372/agent/osx109/master/centos7/pe_acceptance_tests/acceptance/tests/pe_checks/spot_check_lm.rb:19
    /var/lib/jenkins/workspace/qe_beaker_intn-sys_vpool-pe372/agent/osx109/master/centos7/lib/beaker/dsl/structure.rb:58
    /var/lib/jenkins/workspace/qe_beaker_intn-sys_vpool-pe372/agent/osx109/master/centos7/pe_acceptance_tests/acceptance/tests/pe_checks/spot_check_lm.rb:1
    /var/lib/jenkins/workspace/qe_beaker_intn-sys_vpool-pe372/agent/osx109/master/centos7/lib/beaker/test_case.rb:133
    /var/lib/jenkins/workspace/qe_beaker_intn-sys_vpool-pe372/agent/osx109/master/centos7/lib/beaker/test_case.rb:133
    /usr/local/rvm/rubies/ruby-1.9.3-p484/lib/ruby/1.9.1/benchmark.rb:295
    /var/lib/jenkins/workspace/qe_beaker_intn-sys_vpool-pe372/agent/osx109/master/centos7/lib/beaker/test_case.rb:130
    /var/lib/jenkins/workspace/qe_beaker_intn-sys_vpool-pe372/agent/osx109/master/centos7/lib/beaker/test_suite.rb:318
    /var/lib/jenkins/workspace/qe_beaker_intn-sys_vpool-pe372/agent/osx109/master/centos7/lib/beaker/test_suite.rb:315
    /var/lib/jenkins/workspace/qe_beaker_intn-sys_vpool-pe372/agent/osx109/master/centos7/lib/beaker/test_suite.rb:315
    /var/lib/jenkins/workspace/qe_beaker_intn-sys_vpool-pe372/agent/osx109/master/centos7/lib/beaker/test_suite.rb:364
    /var/lib/jenkins/workspace/qe_beaker_intn-sys_vpool-pe372/agent/osx109/master/centos7/lib/beaker/cli.rb:171
    /var/lib/jenkins/workspace/qe_beaker_intn-sys_vpool-pe372/agent/osx109/master/centos7/lib/beaker/cli.rb:99
    /var/lib/jenkins/workspace/qe_beaker_intn-sys_vpool-pe372/agent/osx109/master/centos7/bin/beaker:6
    /var/lib/jenkins/workspace/qe_beaker_intn-sys_vpool-pe372/agent/osx109/master/centos7/pe_acceptance_tests/vendor/bundle/ruby/1.9.1/bin/beaker:23
    /var/lib/jenkins/workspace/qe_beaker_intn-sys_vpool-pe372/agent/osx109/master/centos7/pe_acceptance_tests/vendor/bundle/ruby/1.9.1/bin/beaker:23
```

**Comment by er0ck** [comment]
@tvpartytonight FYI: kicked it and the osx109 cell passed.

**Comment by tvpartytonight** [comment]
@er0ck thanks for all your work for this PR. LGTM! 
skipping glee

**Pull request merged**